super-team-picker/views/show.ejs

```diff
@@ -0,0 +1,78 @@
+
+<%- include("./partials/header") %> 
+
+
+<div class="card">
+    <div class="card-body">
+        <h4 class="card-title"><img src="https://images.unsplash.com/photo-1518492104633-130d0cc84637?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=926&q=80" alt="" class="img-circle team-logo rounded-circle mr-3 small" height="72px" width="72px"><%= cohort.title %> <%= cohort.id %></h4>
```

Comment:
An oven baked chicken image is used on each show page instead of the cohort's logo 🍗